stock_available_to_promise_release/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Stock Available to Promise Release",
-    "version": "14.0.2.4.0",
+    "version": "14.0.2.5.0",
```

Comment:
It's the job of the OCA bot to increase this version number, unless you have a migration script to play for what you ship

Reply:
Shouldn't this be as considered as minor change? Also if there is no migration?

Reply:
From function point of view and as existing configuration can be impacted, it is for me a major change